packages/sdk-components-react/src/html-embed.tsx

```diff
@@ -4,10 +4,15 @@ import {
   useEffect,
   useRef,
   type ForwardedRef,
+  useSyncExternalStore,
+  useState,
+  type ReactNode,
 } from "react";
 import { mergeRefs } from "@react-aria/utils";
 import { ReactSdkContext } from "@webstudio-is/react-sdk";
 
+export const SCRIPT_PROCESSED_TEST_ID_PREFIX = "client-";
```

Comment:
I would like to standardize everytthing tests related being exported under the same name, e.g.

export const __testing__ = {
  scriptTestIdPrefix ....
}

We are often exporting things that are only useful for testing, so this convention would be practical